utilities/decrypt_data/decryption_for_program_service.py

```
@@ -0,0 +1,127 @@
+CONFIG = {
+    "db": {
+        "dsn": ""
+    },
+    "table": "ifix.eg_program_disburse" + "_decpt",
+    "api_url": "http://program-service.ifix:8080/ifms/program-service/v1/disburse/_search",
+    "log_dir": "logs"
+}
+
+# ——— Prepare log directory & filename ——————————————————————————————
+os.makedirs(CONFIG["log_dir"], exist_ok=True)
+ts = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)